packages/sources/peregrine-fund-admin/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Also need to modify packages/sources/tsconfig.json

Reply:
Updated